addons/l10n_fr/data/l10n_fr_demo.xml

```diff
@@ -33,4 +33,11 @@
         <value eval="[ref('base.user_root'), ref('base.user_admin'), ref('base.user_demo')]"/>
         <value eval="{'company_ids': [(4, ref('base.demo_company_fr'))]}"/>
     </function>
+
+    <function model="account.chart.template" name="try_loading">
```

Comment:
Did I missed out something here? the diff seems a bit unusual here

Reply:
It has been moved to `l10n_fr_account`
https://github.com/odoo-dev/odoo/commit/67b4ec4ba431adb4090aa8629d2a556dbf8df54a#diff-0bae8d388be53268c59768e4598b628c1efea56828d2f14b9ea1258268501600

Reply:
Thanks. I have made the change for it.